examples/bindings/sparse_matrix_how_to.py

```diff
@@ -96,10 +95,22 @@
 s.fromDense(numpy.random.random((4,4)))
 print '\nfromDense\n', s
 
-# A sparse matrix can be pickled:
-cPickle.dump(s, open('sm.txt', 'wb'))
-s2 = cPickle.load(open('sm.txt', 'rb'))
-print '\nPickling\n', s2
+# A sparse matrix can be serialized:
+schema = SM32.getSchema()
+
+# Save
+with open("sm.bin", "wb") as f:
+  proto = schema.new_message()
+  s.write(proto)
+  proto.write(f)
```

Comment:
If you open the file in"w+b" mode then you can do "f.seek(0)" after write and then read from the same file. Doesn't really matter though